tls/s2n_security_policies.c

```diff
@@ -1131,6 +1131,18 @@ const struct s2n_security_policy security_policy_rfc9151 = {
     .certificate_preferences_apply_locally = true,
 };
 
+const struct s2n_security_policy security_policy_20250211 = {
```

Comment:
A comment might help here-- what is the security policy supposed to represent? Is this a version of default_tls13 with stricter cipher preferences in line with rfc9151? Or something else?